Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
I don't really want to inline the actual `make` steps. It'll just confuse future contributors and is only an option we ever really deliberately exercise if upstream doesn't provide a make system at all.

I don't have a massively strong preference on whether we use a patch or an `inreplace` but it should be one of those options.